README.md

```diff
@@ -18,6 +18,7 @@ This set of extensions allows you to interact with some of the AWS Services name
  * SNS
  * SQS
  * SSM
+ * Cognito Identity Provider
```

Comment:
Maybe add it at the beginning? We kept the alphabetical order here.

Reply:
Fixed